AssetMappr/presentation/layout.py

```diff
@@ -35,22 +35,25 @@
 # =============================================================================
 # Function
 # =============================================================================
+
+
 def makeLayout(df, master_categories):
 
     return html.Div([
-        
+
         # Layout is split into two overall tabs
         dcc.Tabs([
 
             # Tab 1: Page to view, rate, and upload assets
-            dcc.Tab(label='Tell us about your community', children=[
-                
+            dcc.Tab(id='tab1', label='Tell us about your community', children=[
+
                 # This tab uses a grid structure of rows and columns
-                
+
                 # Row 1
                 dbc.Row([
                     dbc.Col(
-                        html.H4('Use the map below to view and select assets'),
```

Comment:
Could we align this text left over the map? Right now the alignment is a bit off